app.rb

```diff
@@ -0,0 +1,75 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative 'lib/database_connection'
+require_relative 'lib/peep_repository'
+require_relative 'lib/user_repository'
+
+DatabaseConnection.connect('chitter_test')
+
+class Application < Sinatra::Base
+ 
+  use Rack::Session::Pool
```

Comment:
This looks good. I haven't used it but Michal just explained it works for a user's session.